inst/include/interface/interface.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef INST_INCL_INTERFACE_INTERFACE_HPP
```

Comment:
This is just a question about include folder, are we going to put header files to the inst/include or there will be another include folder inside of src folder to sperate .h and .cpp files?

Reply:
The structure in the FIMS R package follows R package development best practices:
- [inst](https://r-pkgs.org/inst.html#inst): everything within gets copied to top level firectory when package installed. 
- [inst/include](https://r-pkgs.org/src.html?q=inst/include#cpp-export): file structure for header files, `.h`
- [src](https://r-pkgs.org/src.html): holds .cpp files that need compiling 
- All header files will go in inst/include so there is no need to add another include folder in src.